demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -1263,8 +1263,28 @@ def _set_integration_params(
             self._change_placeholders_to_values(server_url, conf)
             for conf in integration_params
         ]
-
-        if integration_params:
+        if self.name == "Core REST API":
+            if IS_XSIAM:
+                self.configuration.params = {  # type: ignore
```

Comment:
why ignore?

Reply:
self.configuration is of type `Optional[IntegrationConfiguration]` so mypy warns for AttributeError. Through the code `# type: ignore` is used to ignore this warning.

Reply:
self.configuration is of type `Optional[IntegrationConfiguration]` so mypy warns for AttributeError. Through the code `# type: ignore` is used to ignore this warning.